src/RoslynInsertionTool/scripts/VS-Insertion.ps1

```diff
@@ -61,10 +61,11 @@ Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -
 Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "release/dev16.1-vs-deps"          -toBranch "rel/d16.1" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
 
 # Dev 16.2 Preview 1
+Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "release/dev16.2-preview1-vs-deps"  -toBranch "rel/d16.2" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
 Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "dev16.2-preview1"          -toBranch "rel/d16.2" -insertCore "true"     -queueValidation "true" -dropPath "server"
 Do-Insertion -component "Live Unit Testing" -queueName "TestImpact-Signed"     -fromBranch "dev16.2-preview1"          -toBranch "rel/d16.2" -insertCore "false"    -insertDevdiv "false"   -queueValidation "true"
 
 # Dev 16.2 Preview 2
-Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "master-vs-deps"  -toBranch "lab/d16.2stg" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
+Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "master-vs-deps"  -toBranch "master" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
 Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "master"          -toBranch "lab/d16.2stg" -insertCore "true"     -queueValidation "true" -dropPath "server"
```

Comment:
FYI @shyamnamboodiripad 16.2 Preview 2 is `master` on the VS side until 5/15 (not `lab/d16.2stg`).

Reply:
@dpoeschl Ah I see. I thought we were already in QB mode for Preview 1 at this point - which would mean that Preview 1 changes now need to go into rel/d16.2rel. If so, what is lab/d16.2stg branch going to be used for between now and 5/15?

cc @AbhitejJohn